README.md

```diff
@@ -30,6 +30,7 @@ Check back with us to see new additions and improvements - and please don't hesi
 | Account Member       |       &check;       |                   |                 |
 | Account Role         |       &check;       |                   |                 |
 | Account              |       &check;       |      &check;      |     &check;     |
+| Block                |                     |      &check;      |     &check;     |
```

Comment:
I'm going to try to find a list of objects in Prefect, ideally one that delineates which are Cloud-only. We may want to make that distinction somewhere in our TF provider docs.

Reply:
hah i was just thinking that. from this list, i think we can say:

Cloud-only:
- Account Member
- Account Role
- Account
- Service Account
- Team
- Workspace Access
- Workspace Role
- Workspace

Cloud and OSS
- Block
- Variable
- Work Pool

Reply:
Sweet, thanks - updated in 4daaabc.